users/models.py

```diff
@@ -8,3 +9,24 @@ class Person(models.Model):
 
     class Meta:
         abstract = True
+
+
+class Coach(Person):
+    """
+    Coaches manage the registration of participants (i.e. participants
+    don't register themselves.
+    """
+    club = models.ForeignKey('Club', null=True)
+    phone_number = PhoneNumberField()
+    email = models.EmailField(max_length=255)
+
+
+class MainCoach(Coach):
```

Comment:
ik zou dit oplossen door de boolean die je hier hebt, gewoon toe te voegen aan de coach class. Die dan een default te geven dat ie false is. Aparte class lijkt me hier geen toegevoegde waarde hebben.